index.html

```diff
@@ -8,12 +8,129 @@
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+     
 </head>
 <body>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <!-- Add your  markup here -->
+    <!-- Remember: Use semantic  tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+   <!--header-->
+    <header>
+        <img src="img/karma-logo.svg"
+        alt="karma logo" class="logo"/>
+      
+     <nav>
+         <ul>
+          <li>
+              <a href="meetkarma">
+              <strong>Meet Karma</strong></a>
+            </li>
+          <li>
+              <a href="howitworks">How it Works</a>
+            </li>
+          <li>
+              <a href="store">Store</a>
```

Comment:
Add a link the store page in the top menu. The other href URLs should be set as "#"
```suggestion
                 <a href="store.html">Store</a>
```